js/about/newtab.js

```diff
@@ -96,6 +96,7 @@ class NewTabPage extends React.Component {
     }).size > 0
   }
   isBookmarked (siteProps) {
+    // XXX: Fixme, not passing state in!
```

Comment:
i think it's called once per site icon shown on newTab, when you open a new tab.
it seems to be pretty fast.